Source/Peripheral.swift

```diff
@@ -505,6 +520,14 @@ public class Peripheral {
 		return peripheral.rx_didUpdateName.map { return (self, $0) }
 	}
 
+    /**
+     Emits next when it's disconnected.
+     - Returns: Observable which emits next events when `peripheral` was disconnected.
+     */
+    public func monitorDisconnection(peripheral: Peripheral) -> Observable<Peripheral> {
```

Comment:
I've figured out that we have method that does the thing now. So i'm just deleting it.